tools/romimg/src/romimg.c

```diff
@@ -139,12 +143,7 @@ static int GetExtInfoStat(const struct ROMImgStat *ImageStat, struct RomDirFileF
 int CreateBlankROMImg(const char *filename, ROMIMG *ROMImg)
 {
 	unsigned int CommentLength;
-	char LocalhostName[32], cwd[128];
-#if defined(_WIN32) || defined(WIN32)
-	char UserName[32] = "";
-#else
-	char* UserName;
-#endif
+	char LocalhostName[32] = "\0", cwd[MAX_PATH] = "\0", UserName[32] = "\0";
```

Comment:
I recommend to initializate them with `= {0}` instead, it is equivalent to put all the fields with `\0` directly